### PR TITLE
Updating for release notes and ssl_id session persistence

### PR DESCRIPTION
**Pull request by the2hill**

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 53 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-36a534a27/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by rackernexus** [comment]
Build 54 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-40af5ba58/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by cyrichardson** [comment]
@the2hill Would you like for me to provide an editorial review of your changes and help get these published? What is the date you'd like to have these published externally? Thanks!

**Comment by cyrichardson** [comment]
Would you like for me to merge this and request a production publishing build from the folks with the creds to do that?

**Comment by the2hill** [comment]
My apologies, I didn't see the responses here. The date was pushed back and I've added some updates. If you don't mind verifying I would appreciate it! I want to aim for releasing these the morning of the 18th. Thank you!

**Comment by rackernexus** [comment]
Build 55 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-74fe32e87/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by cyrichardson** [comment]
@the2hill No worries! Reviewing now!

File: api-docs/cloud-load-balancers-v1/api-reference/sessions.rst [hunk 1]
```diff
@@ -38,6 +38,10 @@ Table. Session persistence modes
 |              | back-end node. This is supported for HTTPS pass-through and         |
 |              | non-HTTP load balancing only.                                       |
 +--------------+---------------------------------------------------------------------+
+| SSL_ID       | A session persistence mechanism which sends all SSL traffic with    |
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/sessions.rst]: mechanism that sends

File: api-docs/cloud-load-balancers-v1/api-reference/sessions.rst [hunk 2]
```diff
@@ -60,6 +64,11 @@ concurrently, choose one of the following options:
    load balancer supports session persistence for both HTTP and HTTPS
    requests concurrently.
 
+.. note::
+    - SSL session ID persistence (SSL_ID) is not appropriate for application-level session persistence because many SSL clients regularly
+      renegotiate their SSL session ID. To achieve application-level session persistence you should either use IP-based session persistence,
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/api-reference/sessions.rst]: session persistence, you should

File: api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-26-23-20190911.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.. version-v1.26.17-release-notes:
+
+v1.26.23, Sep 18th, 2019
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+What's new
+-----------
+- Updated Atlas Backend Adapter to a more fault tolerant interface.
```
Comment (cyrichardson) [inline api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-26-23-20190911.rst]: Remove period at end of line 8 (to be consistent with what you have in Resolved issues and in other release notes).

**Comment by the2hill** [comment]
Thank you! Updated. I'll get back you next week!

**Comment by rackernexus** [comment]
Build 56 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-b94296345/docs/cloud-load-balancers/v1/). valid for 14 days

**Comment by the2hill** [comment]
@cyrichardson Good morning! I think we're ready to move these out when you get a chance!

**Comment by cyrichardson** [comment]
@the2hill reviewing and merging shortly. Thanks!

**Comment by the2hill** [comment]
Thank you!

**Comment by cyrichardson** [comment]
@the2hill  I believe  new automation was added to publish externally. I'm checking on it.

**Comment by cyrichardson** [comment]
build me

**Comment by cyrichardson** [comment]
@the2hill Working with others to get the external publishing build to work. I will let you know when it succeeds.

**Comment by cyrichardson** [comment]
@the2hill Build published thanks to others pushing things about!! 
https://developer.rackspace.com/docs/cloud-load-balancers/v1/release-notes/#v1-26-23-sep-18th-2019

**Pull request merged**